qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -11,12 +11,6 @@
 
 class TestCephFSShell(CephFSTestCase):
     CLIENTS_REQUIRED = 1
-    py_version = 'python'
-
-    def setUp(self):
-        CephFSTestCase.setUp(self)
```

Comment:
This is call is required since it does a lot of basic stuff including initializing mount_a.

Reply:
The parent class `CephFSTestCase` will implicitly be called if `setUp` is not present in the child.

Reply:
Ok.